front-end-web/src/Components/Layouts/NavBar.tsx

```diff
@@ -1,21 +1,102 @@
 import React, { Fragment } from "react";
 import { Props } from "../../@Types";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import MenuIcon from "@material-ui/icons/Menu";
+import AccountCircle from "@material-ui/icons/AccountCircle";
+import MenuItem from "@material-ui/core/MenuItem";
+import Menu from "@material-ui/core/Menu";
 
 /**
  * TODO: Add Documentation
  */
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1
+    },
+    menuButton: {
+      marginRight: theme.spacing(2)
+    },
+    title: {
+      flexGrow: 1
+    }
+  })
+);
+
 const NavBar: React.FunctionComponent<Props> = ({
   state,
   handlers,
   classes
 }) => {
+  const [auth, setAuth] = React.useState(true);
```

Comment:
You should be able to remove line 36 now, this is handled by the state parameter being passed into the element.